app/controllers/user_controller.rb

```diff
@@ -0,0 +1,79 @@
+class UserController < ApplicationController
```

Comment:
You should not be defining this. In with Gem assignment you are expected to use gems to do user management and document management

Reply:
Moved this logic device gem package